core/geometry/src/topology/Triangulation.ts

```diff
@@ -177,25 +177,9 @@ export class Triangulator {
     } else {
       for (const p of interior) {
         context.insertAndRetriangulate(p, true);
-        numInsert++;
-        if (numInsert > 16) {
-          /*
-          context.reset();
-          Triangulator.flipTriangles(context.graph);
-          // console.log (" intermediate flips " + numFlip);
-          */
-          numInsert = 0;
-        }
+        numInsert++; // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
wht did you keep `numInsert`?

Reply:
Indispensable for debugging!  Harmless to keep.